### ISSUE TITLE
Use a single VLC player instead of one per window

### ISSUE THREAD

**Issue by Cimbali** [Cimbali/pympress#46]
To save resources and guarantee a perfect sync between the video in both windows, we should use a single player instance and split the output in both windows.

It does not seem possible out of the box. Relevant topics on the VLC forums:
- [Rendering video in two separate windows.](https://forum.videolan.org/viewtopic.php?t=128598)
- [displaying one video in multible windows](https://forum.videolan.org/viewtopic.php?f=32&t=137559)

So basically extend libVLC and/or the python plugin, to use the `clone` splitter and embed the second output.
